app/gzac/src/main/resources/config/documenten-api/leningen.zgw-document-list-column.json

```diff
@@ -0,0 +1,16 @@
+{
+    "changesetId": "leningen.zgw-document-list-columns-v1",
+    "case-definitions": [
+        {
+            "key": "leningen",
+            "columns": [
```

Comment:
There's code in the `DocumentenApiColumnDeploymentService` that should be updated with the introduction of autodeployment. 

Currently, when a document definition is deployed, the default DocumentenApiColumns are set for that document definition. This should only happen, however, if there's no applicable `zgw-document-list-column` files. This requires looking into the contents of the file.